content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+
+.. sourcecode:: rst
+
+    .. attention::
+    On Linux and Unix, this authentication method can only be used when the
+    SFTPPlus service is started as `root`.
+
+
+Header formats
+--------------
+
+- Heading 1 - #
+- Heading 2 - =
+- Heading 3 - -
+- Heading 4 - ^
+
+
+Embedding Screenshots and Images
+--------------------------------
+
+Ensure screenshots are updated, legible, take up the screen width and any
+commands or settings are correct.
+
+.. sourcecode:: rst
+
+    .. image:: /_static/guides/image.png
+        :alt: Description of the image
+        :align: center
+
+
+Internal and external linking
+-----------------------------
+
+When linking to internal documentation pages, use the :doc: tag:
+
+.. sourcecode:: rst
+
+    :doc:`link to Local Manager</operation/local-manager>` `` 
+
+When linking to internal sections within a page, use the :ref: tag:
+
+.. sourcecode:: rst
+
+    :ref:`section in this page <internal-page-link>`
+
+For the :ref: link, create an anchor to the section:
+
+.. sourcecode:: rst
+
+    ._internal-page-link:
+
+When linking to external web links:
+
+.. sourcecode:: rst
+
+    `Bug Writing Guidelines <http://developer.mozilla.org/en/docs/Bug_writing_guidelines>`_
+
+
+Breaking up long lines of logs
+------------------------------
+
+Add a pipe (|) to break up a long log line such as below:
+
+.. sourcecode:: rst
+
+    | 20182 2017-01-30 11:56:41 Process user 127.0.0.1:50568 Account "jan"
+      logged in.
+
+
+Describing a configuration option
+---------------------------------
+
+Example:
+
+.. sourcecode:: rst
+
+    :Default value: 'DEFAULT-EXAMPLE'
+    :Optional: No/Yes
+    :From version: VERSION_HERE
+    :Values: * The values section should only list the type of values supported
+             * Examples include Path, Disabled, Inherit, Path+${USER}
+    :Description:
+        The description further describes the configuration options for the
+        user and what is expected.
+
+
+Adding configuration and log examples
+-------------------------------------
+
+Examples of configuration or logs in the documentation should be edited to be
+more of a real world example. 
+
+For example, instead of 'user', add a real name such as 'alice' or 'bob':
+
+.. sourcecode:: rst
+
+    [accounts/mark-uuid]
+    name = mark
+    enabled = Yes
+    type = application
+    group = Staff
+    description = Staff SFTPPlus application account for Mark
+    home_folder_path = /PATH/TO/MARK/HOME
+    password = PASSWORD
+
+
+Updating the documentation
+--------------------------
+
+The release process will be similar to the rest of the software.
+
+An empty ``.ignore`` with the internal ID can be added in the release notes.
+
+Further details about generating and building documentation is found in the
+chevah server repository.
+
+When creating a new page, ensure to add the page name in the index.rst so that
+the page appears in the index documentation tree.
+
+
+Docstrings
```

Comment:
maybe rename it to "Documenting the code"

docstrings are just a way to document the code... and tests or examples can also serve as documentation for the code